### PR TITLE
Add navigation

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: modules/layout/app-bar.jsx [hunk 1]
```diff
@@ -9,6 +9,25 @@ const SearchAppBar = ({ children, className, ...restProps }) => (
...
+            Data providers
+          </Button>
+        </li>
+        <li>
+          <Button tag="a" href="/" className={styles.link}>
+            Services
+          </Button>
+        </li>
+        <li>
+          <Button tag="a" href="/" className={styles.link}>
```
Comment (viktor-yakubiv) [inline modules/layout/app-bar.jsx:25]:
```suggestion
          <Button tag="a" href="/about" className={styles.link}>
```

File: modules/layout/app-bar.jsx [hunk 2]
```diff
@@ -9,6 +9,25 @@ const SearchAppBar = ({ children, className, ...restProps }) => (
...
     {children}
+    <AppBar.Item tag="nav" className={styles.navigation}>
+      <ul className={styles.navigationList}>
+        <li>
+          <Button tag="a" href="/" className={styles.link}>
+            Data providers
+          </Button>
+        </li>
+        <li>
+          <Button tag="a" href="/" className={styles.link}>
```
Comment (viktor-yakubiv) [inline modules/layout/app-bar.jsx:20]:
```suggestion
          <Button tag="a" href="/services" className={styles.link}>
```

File: modules/layout/app-bar.jsx [hunk 3]
```diff
@@ -9,6 +9,25 @@ const SearchAppBar = ({ children, className, ...restProps }) => (
       <Logo />
     </AppBar.Brand>
     {children}
+    <AppBar.Item tag="nav" className={styles.navigation}>
+      <ul className={styles.navigationList}>
+        <li>
+          <Button tag="a" href="/" className={styles.link}>
```
Comment (viktor-yakubiv) [inline modules/layout/app-bar.jsx:15]:
```suggestion
          <Button tag="a" href="/data-providers" className={styles.link}>
```

**Review by viktor-yakubiv — APPROVED** [review]
Good! 👍

I suggested relative links. Feel free to convert to absolute adding `//core.ac.uk` if you wish. I am not sure what's better.

Thank you for fixing this 🙂

**Comment by Joozty** [comment]
Oh I forgot to add href. 😄 Thanks for review

**Comment by viktor-yakubiv** [comment]
By the way, GitHub has a function 'add the suggestion to bulk commit'. I've never used but this may be useful 🙂

**Pull request merged**